modules/instance_template/main.tf

```diff
@@ -18,6 +18,13 @@
 # Locals
 #########
 
+data "google_service_account" "existing_sa" {
+  provider   = google-beta
+  count      = local.create_service_account == false ? 1 : 0
+  account_id = element(split("@", google_compute_instance_template.tpl.service_account[0].email), 0)
+  project    = element(split("@", element(split(".", google_compute_instance_template.tpl.service_account[0].email), 0)), 1)
```

Comment:
Do you need to provide project? I think just account ID should work.

Reply:
Great catch. Thanks!